SwrveSDK/src/main/java/com/swrve/sdk/conversations/engine/model/Content.java

```diff
@@ -1,6 +1,11 @@
 package com.swrve.sdk.conversations.engine.model;
 
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 public class Content extends ConversationAtom {
+    public static final String YOUTUBE_VIDEO_ID_REGEX = "^https?://.*(?:youtu.be/|v/|u/\\w/|embed/|watch?v=)([^#&?]*).*$";
```

Comment:
Indeed. It didn't cost anything to add the youtu.be to the SDKS. So if this guy comes in https://swrvedev.jira.com/browse/SWRVE-9267 and we decide to support him, there'll be no extra Android work needed :)

So we don't have to officially support it but I wanted to make sure it was robust enough.